### DISCUSSION TITLE
dwsim 8.8.2 crash using recycle

### DISCUSSION THREAD

**Discussion by ML427** [DanWBR/dwsim#680] [General]
Hi everyone,

I recently downloaded DWSIM version 8.8.2, and I've encountered a problem. When I insert a recycle unit and try to select it with the mouse, the program crashes.

Has anyone else experienced this issue? If so, could you please share how you resolved it? Any suggestions or workarounds would be greatly appreciated!

Thank you!

**Comment by DanWBR** [comment]
@ML427 what operating system and user interface (Classic UI or Cross-Platform UI)?

**Reply by ML427** [reply]
Thanks for the reply. I was on Linux, classic dwsim UI. I solved the problem using the microsoft version, where this does not seem to happen.
